### PR TITLE
Localize more dates and texts

### PR DESCRIPTION
**Pull request by sephrat**

### CODE REVIEW

**Comment by sephrat** [comment]
BTW, is /frontend/src/pages/Admin/Backup/BackupCard.vue obsolete? It looks a lot like AvailableBackupCard and I couldn't find its uses.

File: frontend/src/utils/index.js [hunk 1]
```diff
@@ -56,22 +17,6 @@ export default {
     const uniqueKey = `${item}-${index}`;
     return uniqueKey;
   },
-  getDateAsText(dateObject) {
```
Comment (hay-kot) [inline frontend/src/utils/index.js:59]: Are you sure these don't do anything? Last time I tried to remove them, I remember them being used to do something in the Meal Plan components.

Comment (sephrat) [reply frontend/src/utils/index.js:59]: They used to, but I made sure I replaced their uses everywhere in favor of $d function. It's a standard function in vue-i18n, more powerful and natively translated.
https://kazupon.github.io/vue-i18n/guide/datetime.html

Comment (hay-kot) [reply frontend/src/utils/index.js:59]: Fancy! Thanks for all work on localization, it makes a huge difference.

**Pull request merged**